deploy/examples/kubernetes/travel-agency/kogito-travels.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: kogito-travels
+spec:
+  envs:
+    - name: KOGITO_DATAINDEX_HTTP_URL
+      value: http://data-index.kogito
+    - name: KOGITO_DATAINDEX_WS_URL
+      value: ws://data-index.kogito
+  replicas: 1
+  image:
+    domain: quay.io
+    namespace: ricardozanini
```

Comment:
maybe update the namespace?
or did you create a image for the travel agency app?

if that is the case, what do you think to create a repo to keep a few examples already built?

Reply:
I've created the image and it's there in my registry.

> if that is the case, what do you think to create a repo to keep a few examples already built?
That's what I've asked before going out on vacations.

I can push these images (also the process-runtimes) to the kiegroup organization on Quay.